<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DmpClientConnectDialog</class>
 <widget class="QDialog" name="DmpClientConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item alignment="Qt::AlignLeft">
         <widget class="QToolButton" name="AddButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QToolButton" name="DeleteButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="SaveButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="Servers">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="2" column="0">
         <widget class="QLabel" name="HostLabel">
          <property name="text">
           <string>Host</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="NameLabel">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="NameEdit">
          <property name="inputMask">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Your personal label for this server</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="HostEdit">
          <property name="placeholderText">
           <string>The server hostname</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="PortLabel">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="PortEdit">
          <property name="placeholderText">
           <string>The server port</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Servers</tabstop>
  <tabstop>NameEdit</tabstop>
  <tabstop>HostEdit</tabstop>
  <tabstop>PortEdit</tabstop>
  <tabstop>AddButton</tabstop>
  <tabstop>DeleteButton</tabstop>
  <tabstop>SaveButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HostEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>inputChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PortEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>inputChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddButton</sender>
   <signal>clicked()</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>addPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeleteButton</sender>
   <signal>clicked()</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>deletePressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Servers</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>selectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DmpClientConnectDialog</sender>
   <signal>accepted()</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>afterAccept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NameEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>inputChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HostEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>hostChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NameEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>nameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PortEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>portChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveButton</sender>
   <signal>clicked()</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>saveToConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputChanged()</slot>
  <slot>addPressed()</slot>
  <slot>deletePressed()</slot>
  <slot>selectionChanged()</slot>
  <slot>afterAccept()</slot>
  <slot>nameChanged(QString)</slot>
  <slot>hostChanged(QString)</slot>
  <slot>portChanged(QString)</slot>
  <slot>saveToConfig()</slot>
 </slots>
</ui>
